1-select-tasks/13.txt

```diff
@@ -1,3 +1,11 @@
 Create a SQL query to count current and discontinued products:
 
-| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
\ No newline at end of file
+| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
+
+
+SELECT COUNT([p].[ProductName]) AS [TotalOfCurrentProducts],
+(SELECT COUNT([p].[ProductName]) 
+FROM [dbo].[Products] [p]
+WHERE [p].[Discontinued] = 1)  AS [TotalOfDiscontinuedProducts]
+FROM [dbo].[Products] [p]
+WHERE [p].[Discontinued] = 0
```

Comment:
You could write this query a bit differently, so that it's easier to read, select doesn't have to have from and where clauses. 
  ```
SELECT 
  (SELECT COUNT([p].[ProductName]) FROM [dbo].[Products] [p] WHERE [p].[Discontinued] = 1) AS [TotalOfCurrentProducts],
  (SELECT COUNT([p].[ProductName]) FROM [dbo].[Products] [p] WHERE [p].[Discontinued] = 0) AS [TotalOfDiscontinuedProducts]
```
I'm going to close this pull request, but when you finish tasks, if you have time try to solve this with [group by ](https://docs.microsoft.com/en-us/sql/t-sql/queries/select-group-by-transact-sql) and [case](https://docs.microsoft.com/en-us/sql/t-sql/language-elements/case-transact-sql):